projectDocs/dev/developerGuide/developerGuide.md

```diff
@@ -747,38 +747,36 @@ Be very careful to keep all tabs and spaces intact.
 
 Once saved in the right place, either restart NVDA or choose Reload Plugins found under Tools in the NVDA menu.
 
-    --- start ---
-    #Window utility scripts for NVDA
-    #Developer guide example 4
-    
-    import globalPluginHandler
-    from scriptHandler import script
-    import ui
-    import api
-    
-    class GlobalPlugin(globalPluginHandler.GlobalPlugin):
+```py
+#Window utility scripts for NVDA
+#Developer guide example 4
+import globalPluginHandler
+from scriptHandler import script
+import ui
+import api
     
```

Comment:
While at it: remove spaces on blank lines to conform with NVDA's linting rules.

For example:

```suggestion

```

But there are much more blank lines that are not empty (i.e. containing spaces)